dev/index.html

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormIt.js"></script>
     <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormItPluginUI.js"></script>
     <link rel="stylesheet" type="text/css" href="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormItPluginStyling.css">
-    <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/FormItInterface.js"></script>
+    <script type="text/javascript" src="https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles/v23_0/FormItInterface.js"></script>
```

Comment:
Should not be https://formit3d.github.io/FormItExamplePlugins/SharedPluginFiles anymore.
Should use the new shared utilities
 https://formit3d.github.io/SharedPluginUtilities/
Are the formitinterface.js files updated there?